### PR TITLE
fix maxline violations i.e over 120 chars

### PR DESCRIPTION
**Pull request by davidmort**

### CODE REVIEW

**Review by rhiGriff — APPROVED** [review]

**Review by benjefferies — APPROVED** [review]

File: collex-loader/load_events.py [hunk 1]
```diff
@@ -97,7 +98,13 @@ def get_collection_exercise_uuid(row, api_config):
     print("Dry run: {}".format(dry_run))
 
     api_config = config['api']
-    event_handler = dump_event if dry_run else partial(post_event, user=api_config['user'], password=api_config['password'], url=api_config['post-url'])
+    if dry_run:
+    	event_handler = dump_event 
+    else: 
+	partial(post_event,
```
Comment (benjefferies) [inline collex-loader/load_events.py:104]: Think this is broken 👎

Comment (andrewmil) [reply collex-loader/load_events.py:104]: yeah this wont run for me anymore

Comment (davidmort) [reply collex-loader/load_events.py:104]: Hmm ok, I'll take a look and fix when I get home

**Comment by rhiGriff** [comment]
I think this was broken before the change I was having errors today with loading events

**Pull request merged**